components/is-analytics-solutions/org.wso2.carbon.sp.is.analytics.solutions/widgets/IsAnalyticsAttemptsOverTime/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "IsAnalyticsAttemptsOverTime",
+  "version": "1.0.0",
+  "private": true,
+  "dependencies": {
+    "@material-ui/core": "^1.4.0",
+    "@wso2-dashboards/widget": "^1.2.2",
+    "css-loader": "^0.28.11",
```

Comment:
Are the loaders needed as direct dependencies? Cant we add them in dev dependency (If this is needed only as peer dependency)?